### ISSUE TITLE
Pagefind files fail to load when using a path prefix

### ISSUE THREAD

**Issue by cooljeanius** [tweetback/tweetback#96]
I recently followed the directions in #13 to work towards fixing cooljeanius/my_tweetback_archive#1. However, even after doing so, the "search" interface still doesn't show up on my archive page; see here: 
https://cooljeanius.github.io/my_tweetback_archive/ 
@stefanbohacek reported noticing this, too. Maybe the path-rewriting code needs some tweaking?

**Comment by bglw** [comment]
Some context in this mastodon thread: https://fosstodon.org/@egallager@treehouse.systems/112128338307861881

tl;dr — Jekyll won't serve the `_pagefind` directory as it leads with an underscore. Can be fixed in usage by configuring Pagefind's bundle directory.

From the Tweetback side of things, migrating the project to Pagefind 1.0 (mainly just a few changed keys) will also change Pagefind's default path from `_pagefind` to `pagefind`, which was a 1.0 change made specifically for this issue 🙂

**Comment by cooljeanius** [comment]
> Some context in this mastodon thread: [fosstodon.org/@egallager@treehouse.systems/112128338307861881](https://fosstodon.org/@egallager@treehouse.systems/112128338307861881)
> 
> tl;dr — Jekyll won't serve the `_pagefind` directory as it leads with an underscore. Can be fixed in usage by configuring Pagefind's bundle directory.
> 
> From the Tweetback side of things, migrating the project to Pagefind 1.0 (mainly just a few changed keys) will also change Pagefind's default path from `_pagefind` to `pagefind`, which was a 1.0 change made specifically for this issue 🙂

Do you have a link to the Pagefind change in question?

**Comment by zachleat** [comment]
I think simply adding a `.nojekyll` file here would solve this problem! The paths are correct!

https://github.blog/2009-12-29-bypassing-jekyll-on-github-pages/

**Comment by cooljeanius** [comment]
> I think simply adding a `.nojekyll` file here would solve this problem! The paths are correct!
> 
> [github.blog/2009-12-29-bypassing-jekyll-on-github-pages](https://github.blog/2009-12-29-bypassing-jekyll-on-github-pages/)

OK, this seems to have worked for my particular usecase, thanks! I think it's still worth keeping this issue open for migrating Tweetback to Pagefind 1.0, though
